com.hoptodesk.HopToDesk.yml

```diff
@@ -0,0 +1,52 @@
+id: com.hoptodesk.HopToDesk
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: hoptodesk
+icon: share/icons/hicolor/scalable/apps/hoptodesk.svg
+appdata:
+  license: FSFAP
+  url: https://hoptodesk.com
+  metadata_license: FSFAP
```

Comment:
```suggestion
```

Reply:
Same here.

Reply:
same. Which documentation say to do that?